source/extensions/bootstrap/reverse_tunnel/common/reverse_connection_utility.h

```diff
@@ -22,6 +22,8 @@ class ReverseConnectionUtility : public Logger::Loggable<Logger::Id::connection>
 public:
   static constexpr absl::string_view PING_MESSAGE = "RPING";
   static constexpr absl::string_view PROXY_MESSAGE = "PROXY";
+  static constexpr absl::string_view kDefaultReverseTunnelRequestPath =
```

Comment:
Envoy code styles are using `CONSTANT_VAR` values, and Google C++ styles recommend `k*`, I am fine either way, but would be good to unify it.